mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/payment/AmountChargeHandler.java

```diff
@@ -151,10 +129,10 @@ public boolean handle(MessageContext context) throws Exception {
 
             if(executor.isUserAnonymization()) {
                 String resourcePath = executor.getSubResourcePath();
-                String urlmsisdn = resourcePath.substring(1, resourcePath.indexOf("transactions") - 1);
-                String unmaskedUrlmsisdn = UserMaskHandler.transcryptUserId(URLDecoder.decode(urlmsisdn, "UTF-8"),
+                String urlMsisdn = resourcePath.substring(1, resourcePath.indexOf("transactions") - 1);
```

Comment:
this part has been duplicated in both AmountRefundHandler.java and AmountChargeHandler.java handlers. So its better if this can be moved to common method

Reply:
Done